eZ/Publish/Core/FieldType/Validator/ImageValidator.php

```diff
@@ -13,15 +13,15 @@
 class ImageValidator extends Validator
 {
     /**
-     * @inheritDoc
+     * {@inheritdoc}
```

Comment:
Is it really needed? I think most tools should already use docblock from parent method.

Reply:
This change was not intended.